packages/aws-cdk-lib/aws-ecr/lib/lifecycle.ts

```diff
@@ -96,3 +96,150 @@ export enum TagStatus {
    */
   UNTAGGED = 'untagged',
 }
+
+/**
+ * Properties for creating a LifecycleRule
+ */
+export interface LifecycleRuleProps extends LifecycleRule {}
```

Comment:
Why does this `extends LifecycleRule` ?

An interface either represents a static bag of data, or it represents props. In this case, `LifecycleRule` is data, so it seems weird to me that `Props` should extend it.